### PR TITLE
Include security section in issue/PR templates

### PR DESCRIPTION
**Pull request by marcokrikke**
Ref. none

## Proposed changes

Improve visibility of security concerns and testing in the templates we use for issues and PRs. 

A suggested improvement from the internal ISO27001 audit.

## Testing

None.

## Documentation

It helps documenting how we treat security concerns during development.

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -8,6 +8,8 @@ Describe the big picture of your changes here.
 
 Describe how you have tested the changes. Did you test the functionality by hand? Have automated tests been created? If not, why not, what issues did you hit creating a unit or integration test?
 
+If applicable, please also describe if there are any security concerns (e.g. mentioned in the related issue) and how these are tested.
+
```
Comment (johanwortel) [inline .github/PULL_REQUEST_TEMPLATE.md:15]: Now it looks like this is an addition to the Testing paragraph. Does not seem correct to me.

Comment (marcokrikke) [reply .github/PULL_REQUEST_TEMPLATE.md:15]: What do you suggest? Do you want to mention any security concerns before the testing paragraph, and then use the testing paragraph to tell how we've tested the security?

Comment (johanwortel) [reply .github/PULL_REQUEST_TEMPLATE.md:15]: That is better. Either a new (optional?) paragraph or an addition to 'Proposed changes'. Some remarks under 'Testing' can be useful, but our development environment is very different from a user's, although many security issues apply to development as well. Reminds me of GitHub workflow issues that also are beyond the limited scope of PR development.

Comment (marcokrikke) [reply .github/PULL_REQUEST_TEMPLATE.md:15]: How about now?

**Review by johanwortel — APPROVED** [review]

**Review by Mario2804 — APPROVED** [review]

**Pull request merged**